index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="container section-1">
```

Comment:
Numerowanie klas nie zawsze jest dobrym pomysłem. Co jeśli zmienimy ich kolejność? W kodzie będzie 3, 2, 4, 1, 5 - będzie to ok? Czy może będzie zmieniać nawy klas. Tak czy inaczej będzie to problematyczne ;)